src/lib/coda_base/insecure.ml

```diff
@@ -15,4 +15,4 @@ let private_key_generation = true
 
 let randomness = true
 
-let vrf_threshold_check = false
+let vrf_threshold_check = true
```

Comment:
Can we move this out of `Insecure`? I'd like to stop using this library, if possible. Seems like this should be a compile time flag since it effects the snark.

(I won't necessarily require this for the PR since you didn't add it, but it would be nice still).